packages/run-protocol/src/vaultFactory/types.js

```diff
@@ -16,7 +16,7 @@
  */
 
 /**
- * @typedef {Object} Rates
+ * @typedef {Object} Rates - XXX loan terms?
```

Comment:
another XXX. I'm not sure what to do with these.

If they are intended to land on master, ideally they would have corresponding issue #s.